Fail to run `react-scripts build`

The lib seems to good for us to call AWS API gateway's APIs :)
But when I try to add React project launched by `create-react-app`, the `react-scripts build` command is always failed...

# Error message

```
Creating an optimized production build...
Failed to compile.

Failed to minify the code from this file: 

 	./node_modules/retryx/index.js:7 

Read more here: http://bit.ly/2tRViJ9
```

# Testing repo
https://github.com/hideokamoto/v3.wp-kyoto.net/tree/try/retryx

# Reproduce

```
$ git clone git@github.com:hideokamoto/v3.wp-kyoto.net.git
$ cd v3.wp-kyoto.net/
$ git checkout try/retryx
Branch try/retryx set up to track remote branch try/retryx from origin.
$ npm i or yarn install
$ npm run build

> wp-kyoto.net@0.1.0 build /Users/dc_hideokamoto/tmp/v3.wp-kyoto.net
> react-scripts build && react-snapshot

Creating an optimized production build...
Failed to compile.

Failed to minify the code from this file: 

 	./node_modules/retryx/index.js:7 

Read more here: http://bit.ly/2tRViJ9

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! wp-kyoto.net@0.1.0 build: `react-scripts build && react-snapshot`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the wp-kyoto.net@0.1.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/dc_hideokamoto/.npm/_logs/2018-01-16T01_44_31_821Z-debug.log
```

Do you know how to resolve the problems ?
Thanks !

--- a/index.js
+++ b/index.js
@@ -1,10 +1,10 @@
 "use strict";
 
-const lib = require("./dist");
+var lib = require("./dist");
 
 module.exports = lib.default;
 
-for (let p in lib) {
+for (var p in lib) {
   if (!module.exports.hasOwnProperty(p)) module.exports[p] = lib[p];
 }
 

